hr_timesheet_sheet_attendance/models/hr_attendance.py

```diff
@@ -0,0 +1,108 @@
+from odoo import api, fields, models, _
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
```

Comment:
@bizzappdev should be `from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT, DEFAULT_SERVER_DATE_FORMAT`